bookkeeper-server/src/main/java/org/apache/bookkeeper/client/SyncCallbackUtils.java

```diff
@@ -17,12 +17,12 @@
  */
 package org.apache.bookkeeper.client;
 
+import static org.apache.bookkeeper.client.LedgerHandle.LOG;
```

Comment:
it is not needed.

Reply:
Thanks. It is originally there, this change adjust the order of importing, will handle it.